docs/tutorials/cloud_fit_deploy/cloud-aws-sagemaker-deployment.md

```diff
@@ -148,17 +162,16 @@ model_data = sagemaker_session.upload_data(
 Prepare transform job:
 
 ```{.python}
-instance_type = "ml.m5.2xlarge"
+instance_type = "ml.m5.2xlarge"  # You might want to use GPU instances for Text/Image/MultiModal Predictors etc
```

Comment:
Could we give an GPU instance example? Maybe a G4 instance.

Reply:
good call